docs/source/install.rst

```diff
@@ -141,11 +141,11 @@ The :file:`/etc/pcocc/batch.yaml` configuration file contains configuration pert
      etcd-protocol: http
      etcd-auth-type: password
 
-If you enabled TLS, select the *https* etcd-protocol and define the **etcd-ca-cert** parameter to the path of the CA certificate created for etcd (see :ref:`Deploy a secure etcd cluster <etcd-production>`).
+If you enabled TLS, select the *https* etcd-protocol and define the ``etcd-ca-cert`` parameter to the path of the CA certificate created for etcd (see :ref:`Deploy a secure etcd cluster <etcd-production>`).
```

Comment:
The formatting is as intended for etcd-ca-cert, however we should have used ``**etcd-protocol**``. We could discuss changing the style for parameter names but this would have to be changed accordingly in the rest of the documentation.